src/components/filterPatterns/popover/__tests__/Index.Test.tsx

```diff
@@ -27,11 +27,12 @@ describe('<PopoverFilter />', () => {
     expect(await screen.findByText('Popover content')).toBeInTheDocument();
   });
 
-  it('should show the reset button if the filter is applied', async () => {
+  it('should show the reset button if the filter is applied and display value is provided', async () => {
```

Comment:
can you please add another test that covers the new behavior?
- isApplied: true + displayValue empty